frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```
@@ -58,11 +81,30 @@ $question-input-width: 337px
     .list-item-wrapper .question-index
         cursor: ns-resize
 
+.drag-container
+    border: 2px solid $light-gray
+    border-radius: 10px
+    margin: 10px
+
+.button-container
+    display: flex
+    align-items: center
+    justify-content: center
+
 .questions-divider
     margin-bottom: 20px
 
 .plus-sign-btn
-    color: $dark-gray
+    border-radius: 24px
+    font-family: 'Inter'
+    font-style: normal
+    font-weight: 400
+    font-size: 14px
+    align-items: center
+    background: $white
+    border: 1px solid $nepal
+    color: $midnight
+    max-height: 36px
```

```suggestion
    border-radius: 24px
    font-family: 'Inter'
    font-weight: 400
    align-items: center
    border: 1px solid $nepal
    max-height: 36px
```

Why do we need these properties? They are by default